prod-devel-rcar.yaml

```diff
@@ -7,6 +7,7 @@ variables:
   DOMD_BUILD_DIR: "build-domd"
   DOMU_BUILD_DIR: "build-domu"
   XT_DOMD_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-domd.dtb"
+  XT_DOMUSER_DTB_NAME: ""
```

Comment:
I checked in sources that DomU's DTB is provided to dom0 by 
```
install -m 0644 ${S}/${XT_DOMU_DTB_NAME} ${D}${libdir}/xen/boot/domu.dtb
```
in https://github.com/xen-troops/meta-xt-common/blob/master/meta-xt-control-domain/recipes-guest/domu/domu.bb#L29
So I'm not sure we need this commit. Also I have tested that DomU successfully runs on prod-devel-rcar without this change (with fixed bl2 and proper domd.cfg).

But we really need to clarify and describe in documentation, which device tree should be provided by the domain's recipes and which have to be added to `XT_DEVICE_TREES`. But this need to be discussed.